api/etcdserverpb/rpc.pb.go

```diff
@@ -1792,7 +1792,9 @@ type SnapshotResponse struct {
 	// remaining_bytes is the number of blob bytes to be sent after this message
 	RemainingBytes uint64 `protobuf:"varint,2,opt,name=remaining_bytes,json=remainingBytes,proto3" json:"remaining_bytes,omitempty"`
 	// blob contains the next chunk of the snapshot in the snapshot stream.
-	Blob                 []byte   `protobuf:"bytes,3,opt,name=blob,proto3" json:"blob,omitempty"`
+	Blob []byte `protobuf:"bytes,3,opt,name=blob,proto3" json:"blob,omitempty"`
+	// version of server that created the snapshot.
```

Comment:
Please warn user that in case of cluster of server's with binaries in different versions, this can differ before calls.

Reply:
Not sure what you mean with "can differ before calls", in case of example you described, we could add information that this is local cluster version and each server could return different response.

Reply:
Yes - I'm asking to make it more explicit in comment, such that no-one messes this with 'cluster version' that is consistent within the cluster.